modules/installation-guide/partials/proc_installing-che-on-openshift-4-using-the-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// installing-che-on-openshift-using-the-operator
+
+[id="installing-{prod-id-short}-on-openshift-4-using-the-cli_{context}"]
+= Installing {prod-short} on OpenShift 4 using the CLI
+
+.Prerequisites
+
+* `oc` is installed on the cluster. See link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#installing-the-cli[Installing the CLI].
+
+* `{prod-cli}` is installed on the cluster. See xref:using-the-chectl-management-tool.adoc[].
+
+* __<namespace>__ represents the project of the target installation.
```

Comment:
That's not necessary since {prod-cli} will create namespace if it doesn't exist.